.github/workflows/publish.yaml

```diff
@@ -72,7 +72,7 @@ jobs:
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
-          repository: cardanofoundation/adahandle-resolver
+          repository: cardanofoundation/cf-adahandle-resolver
```

Comment:
do we need to brand everything? I would be against. Specially if the hope is to get community member to adopt and develop.

Reply:
It is not for branding it is more to distinguish different adahandle-resolver projects in our gh org. When a community member creates a fork they probably use the input field anyways to rename the project.